### ISSUE TITLE
Concatenated RTL-isolates give wrong levels

### ISSUE THREAD

**Issue by poire-z** [fribidi/fribidi#141]
I think I've found another case where FriBiDi's output differs from the UBA-C output:
(all that in a RTL paragraph context).

![image](https://user-images.githubusercontent.com/24273478/75248772-14fd0c00-57d5-11ea-8126-3b3bdabc20c3.png) [bidi_concatenated_rli.txt](https://github.com/fribidi/fribidi/files/4250039/bidi_concatenated_rli.txt)

```
$ ./fribidi --novisual --nopad --showinput --rtl --levels bidi_concatenated_rli.txt
â§PMâ©â§96 / 110â©              => 1 4 4 1 1 4 4 4 4 4 4 4 4 1
                                                 ^ ^ ^ these look wrong
```
(^ crap as I'm on a latin1 terminal)
and I would get as the final display (which feels wrong):  ![image](https://user-images.githubusercontent.com/24273478/75249388-82f60300-57d6-11ea-8133-89c9899f6dfd.png)

vs UBA-C, which gives the expected display:

https://unicode.org/cldr/utility/bidic.jsp?s=%E2%81%A7PM%E2%81%A9%E2%81%A796+%2F+110%E2%81%A9&b=1&u=110&d=2
![image](https://user-images.githubusercontent.com/24273478/75248993-9359ae00-57d5-11ea-825b-b7f99ca40607.png)

Note that this `96 / 110` in a single RLI with nothing before is fine:  ![image](https://user-images.githubusercontent.com/24273478/75249947-8b9b0900-57d7-11ea-9b92-e0c8f6296f66.png) [bidi_rli.txt](https://github.com/fribidi/fribidi/files/4250092/bidi_rli.txt)

```
./fribidi --novisual --nopad --showinput --rtl --levels bidi_rli.txt
â§96 / 110â©                      => 1 4 4 3 3 3 4 4 4 1
```

(Context at https://github.com/koreader/koreader/issues/5359#issuecomment-590826467)

**Comment by dov** [comment]
Thanks a lot! This is the second bug you found in the algorithm, in spite of it passing all the unicode tests! Fixed in commit:12c8d3e

**Comment by poire-z** [comment]
Thank _you_ !
Confirming this commit solves my test case, and my in-context issue :)
